lib/watcher/cpu-profile.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-var agent = require('../agent');
+var agent = require('../agent')();
```

Comment:
Sian, can can you please rebase this branch onto master once https://github.com/strongloop/strong-supervisor/pull/179 merges.